pages/2020/index.rst

```diff
@@ -0,0 +1,111 @@
+
+Trinity College, Dublin
+-----------------------
+
+.. image:: /images/pyastro_logo_150px.png
+   :align: center
+   :width: 150px
+
+The sixth Python in Astronomy Conference will be held at Trinity College,
+Dublin on **20 - 24 April 2020**.
+
+This conference series aims to bring together a wide variety of developers,
+educators and other contributors to Python packages in the context of all
+forms of astronomy. The conference will include unconference sessions,
+presentations, sprints and tutorials. As well as expanding the community around
+astronomical uses of Python, the conference aims to improve collaboration and
+interoperability between Python packages, and to share knowledge on Python
+packages and techniques.
+
+All participants at the conference will be expected to follow the
+`code of conduct </code-of-conduct>`_.
+
+More details will be made available on this page closer to the week of the
+conference.
+
+Applying
+########
+
+A link to the application page will be posted here in the next few weeks.
+
+
+..
+   Proceedings
+   ###########
+
+Program
+#######
+
+Once finalized, the conference schedule will be posted here.
+
+..
+   The conference schedule and related info `can be found here </2020/schedule>`_.
+
+Chat and Social Media
+#####################
+
+We will be using both Matrix and Slack for online chat throughout the week,
+with a bridge to sync messages from one to the other, so people can use
+whichever platform they prefer.
+You can `go here to get an invitation for the Slack team <https://join.slack.com/t/pyastro/shared_invite/enQtNjk4MjMxNjE3Mzk1LWZkMGI1N2QxOTBlODYzNjk4OGIzMGI0MGU5YjQ2M2Y3ZTM1MTI4ZGU5MmM1ZGIyYzgxYjFmOThiZDg0ZGI5YjY>`_,
+or `here to join the Matrix community <https://riot.im/app/#/group/+pyastro:openastronomy.org>`_.
+We will be using this chat throughout the conference for discussions about
+talks, unconferences, etc. and to facilitate remote participation, so we
+encourage everyone to join the chat and to share the links with anyone else
+who is interested in following the conference.
+
+The twitter hashtag for this workshop is `#pyastro20 <https://twitter.com/hashtag/pyastro20>`_,
+and the main twitter account will be `@pyastro20 <https://twitter.com/pyastro20>`_.
+Feel free to use the hashtag before, during, and after the workshop to connect
+with other participants and the wider community.
+
+Remember, when interacting with people either in person or online, you are required
+to follow the `code of conduct </code-of-conduct>`_.
+
+..
+   Livestream and Live Chat
+   ########################
+
+
+Organising Committee
+####################
+
+The scientific organising committee (SOC) consists of:
+
+* Monica Bobra (co-chair)
+* Drew Leonard (co-chair)
+* Will Barnes
+* Clara Brasseur
+* Juan Luis Cano
+* Rebecca Lange
+* Sophie Murray (local liaison)
+
+..
+   Add LOC as well?
+
+To contact the organizers, send an email to python.in.astronomy.soc@gmail.com.
+
+Venue
+#####
+
```

Comment:
The conference will be held in the [Fitzgerald Building](https://goo.gl/maps/RPX9479x4CHjoUFL9) at the School of Physics, Trinity College Dublin. More information/maps can be found [here](https://www.tcd.ie/Maps/) . 

TCD is located in Dublin City centre and is easily accessible from Dublin Airport (DUB), with flight connections to almost all major European and inter-continental cities. From the airport terminals, Dublin city centre can be reached either by bus or by taxi.

**Taxi**
Taxis are readily available outside both terminal buildings. A taxi will take about 20-30 minutes (depending on traffic). The usual fare for a journey to Dublin city centre is approximately €25. 

**Bus**
TCD can be reached by the following bus services, with relevant drop-off locations indicated by red symbols in the map:
- [Aircoach 700](https://www.aircoach.ie) – €7 one way or €12 return. The Aircoach leaves from the airport every 15 min to the city centre (get off a the Grafton St stop - just beside the TCD front gate).

- [Airlink 757](https://airlinkexpress.ie) – €7 one way or €12 return. This service runs every 30 minutes to the city centre (Pearse Street Station stop).

- Dublin Bus 16 – A cheaper alternative is to take the regular bus from the airport that will take about an hour and will cost €3.30. The exact fare needs to be paid directly to the bus driver, alternatively tickets are available for sale at the small markets in the arrival section of the airport (both Terminal 1 and Terminal 2).

The conference venue is also located close to train links at Pearse and Connolly stations. For public transportation, [Leap cards](https://www.leapcard.ie/Home/index.html) can be used on the buses, trains, and trams (Luas) in the city centre.

Reply:
added. Thanks @drsophiemurray!